src/main/resources/assets/mining_dimension/lang/ru_ru.json

```diff
@@ -0,0 +1,5 @@
+{
+  "block.mining_dimension.teleporter": "Телепорт в измерение добычи",
+  "biome.mining_dimension.mining_biome": "Измерение добычи",
+  "message.wrong_dimension": "Телепортатор не работает в этом измерении переведено сообществом модовестник"
```

Comment:
"Teleporter doesn't work in this dimension *translated by modovestnik community*"
Advertisement of his VK community, just like in any other of his "translations"